pkg/controller/restore_controller.go

```diff
@@ -278,6 +285,10 @@ func (controller *restoreController) getValidationErrors(itm *api.Restore) []str
 		validationErrors = append(validationErrors, "BackupName must be non-empty and correspond to the name of a backup in object storage.")
 	}
 
```

Comment:
You should validate the the included/excluded namespaces are a valid set of includes/excludes. See [backup controller](https://github.com/heptio/ark/blob/master/pkg/controller/backup_controller.go#L299)